From 50865af063c2289d288d3aa6fc215db94340d3d3 Mon Sep 17 00:00:00 2001
From: Weiguang Kong <weiguang.kong@nxp.com>
Date: Thu, 28 Sep 2017 12:44:17 +0800
Subject: [PATCH 2614/5242] MLK-16545-1: uapi: mxc_hifi4: add reset command
 for hifi4

commit  6ec23d0c628f75d1ddffdfe84f89013a3e097b76 from
https://source.codeaurora.org/external/imx/linux-imx.git

add reset command declaration into mxc_hifi4.h file,
this command is used to reset hifi4 codec when seeking

Signed-off-by: Weiguang Kong <weiguang.kong@nxp.com>
Reviewed-by: Daniel Baluta <daniel.baluta@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 include/uapi/linux/mxc_hifi4.h |    1 +
 1 file changed, 1 insertion(+)

diff --git a/include/uapi/linux/mxc_hifi4.h b/include/uapi/linux/mxc_hifi4.h
index 4da1700..27a3c4f 100644
--- a/include/uapi/linux/mxc_hifi4.h
+++ b/include/uapi/linux/mxc_hifi4.h
@@ -34,6 +34,7 @@
 #define HIFI4_UNLOAD_CODEC		_IOW(HIFI4_IOC_MAGIC, 5, unsigned int)
 #define HIFI4_GET_PCM_PROP		_IOW(HIFI4_IOC_MAGIC, 6, unsigned int)
 #define HIFI4_SET_CONFIG		_IOW(HIFI4_IOC_MAGIC, 7, unsigned int)
+#define HIFI4_RESET_CODEC		_IOW(HIFI4_IOC_MAGIC, 8, unsigned int)
 
 #define CODEC_MP3_DEC		1
 #define CODEC_AAC_DEC		2
-- 
1.7.9.5

